File: pinyougou-web-manager/src/main/webapp/js/service/userService.js
//服务层
app.service('userService',function($http){
	    	

	//删除
	this.dele=function(ids){
		return $http.get('../user/delete.do?ids='+ids);
	};
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../user/search.do?page='+page+"&rows="+rows, searchEntity);
	};
	this.updateStatus = function(ids,status){
		return $http.get('../user/updateStatus.do?ids='+ids+"&status="+status);
	}
});

File: pinyougou-interface/src/main/java/cn/itcast/core/service/UserService.java
package cn.itcast.core.service;

import cn.itcast.core.pojo.user.User;
import entity.PageResult;
import org.apache.ibatis.annotations.Select;

public interface UserService {
    void sendCode(String phone);

    void add(String smscode, User user);

    int findCount();

    PageResult search(Integer page, Integer rows, User user);

    User findOne(Long id);

    int findActiveUser();
}
